### PR TITLE
ft(create):create post with an image

### PR DESCRIPTION
**Pull request by ShemaM**

### CODE REVIEW

File: server/src/config/db.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import mongoose from "mongoose";
+import dotenv from "dotenv";
+import { config } from "dotenv";
+import asyncHandler from "../middlewares/async";
+
+config();
+
+const { MONGO_URI, NODE_ENV, MONGO_URI_TEST } = process.env;
+console.log(process.env.NODE_ENV);
```
Comment (habinezadalvan) [inline server/src/config/db.js:9]: Please remove this unnecessary console.log

File: server/src/_data/blog.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+[
+    {
```
Comment (habinezadalvan) [inline server/src/_data/blog.json:2]: Please remove these dummy data

File: server/src/controllers/blog.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import uploader from "../config/cloudinary";
+import Post from "../modal/blog";
+
+class blog {
```
Comment (habinezadalvan) [inline server/src/controllers/blog.js:4]: I think it's better to start classes with Capital letter

File: server/src/modal/blog.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import mongoose from "mongoose";
+
```
Comment (habinezadalvan) [inline server/src/modal/blog.js:2]: I would suggest creating a validating middleware.

**Review by habinezadalvan — CHANGES_REQUESTED** [review]
Please work on the comments below.

**Review by habinezadalvan — COMMENTED** [review]
work on the comments

**Review by habinezadalvan — COMMENTED** [review]
Work on the comment below

**Review by habinezadalvan — COMMENTED** [review]
Work on the comments below

File: server/src/controllers/blog.js (hunk 3 above)
Comment (ShemaM) [reply server/src/controllers/blog.js]: yeah i have started class with capital letter

File: server/src/_data/blog.json (hunk 2 above)
Comment (ShemaM) [reply server/src/_data/blog.json]: all dummy data removed

File: server/src/modal/blog.js (hunk 4 above)
Comment (ShemaM) [reply server/src/modal/blog.js:2]: i have used joi validator

File: server/src/config/db.js (hunk 1 above)
Comment (ShemaM) [reply server/src/config/db.js]: all unnecessary console.log i removed them

**Pull request merged**